zuko/transforms.py

```diff
@@ -30,8 +31,7 @@
 
 from textwrap import indent
 from torch import BoolTensor, LongTensor, Size, Tensor
-from torch.distributions import Transform, constraints, Distribution
-from torch.distributions.transforms import *  # noqa: F403
```

Comment:
Keep this line. It allows users to import any torch transform from `zuko.transforms`, which is very convenient. This is not a safety hazard because F405 (undefined variable or defined from star imports) is not allowed within our code base.

Reply:
Sorry, was not intended. Reverted it